pycon/templates/finaid/email/applicant/request_more/subject.txt

```diff
@@ -0,0 +1 @@
+{% load review_tags %}Message from {{ user.get_full_name|bleach|safe }} requesting more assistance.
```

Comment:
``review_tags`` don't seem to be used anywhere. (Ditto for the overridden subjects in the views file.)